json-glib/plan.sh

```diff
@@ -0,0 +1,31 @@
+pkg_name=json-glib
+pkg_origin=core
+pkg_version="1.2.8"
+pkg_description="..."
```

Comment:
How about "A library for reading and parsing JSON using GLib and GObject data types and API." (pulled from the website)?

Reply:
Oh, I just forgot this one actually ;) thanks!